app/assets/stylesheets/application.scss

```diff
@@ -26,21 +26,21 @@ $hmrc-assets-path: "/coronavirus-job-support-scheme-calculator/assets/lib/hmrc-f
   line-height: 1.09;
 }
 
-.furlough-heading {
+.absent-heading {
```

Comment:
why did you choose this name?

Reply:
Furlough is essentially a leave of absence, so went with absent to keep it to one word.

Reply:
hmm, absent word looks weird though... may be you can use `grant` or something

Reply:
Alright, I'll go with grant